components/toast/theme.css

```diff
@@ -20,29 +21,45 @@
   background-color: var(--toast-background-color);
   bottom: var(--toast-offset);
   display: flex;
-  min-height: var(--toast-min-height);
+  min-width: var(--toast-min-width);
   max-width: var(--toast-max-width);
-  padding: var(--toast-padding);
+  min-height: var(--toast-min-height);
+  padding: var(--toast-spacing);
   position: fixed;
   right: var(--toast-offset);
   transition: all var(--animation-duration) var(--animation-curve-default) var(--animation-duration);
   z-index: var(--z-index-high);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  a.action-link {
+    color: var(--color-teal-light);
+    font-family: var(--font-family-regular);
+    font-size: 14px;
+    letter-spacing: normal;
+    line-height: 19px;
+    text-align: right;
+
+    &:hover {
+      color: var(--color-teal-light);
+    }
```

Comment:
Maybe the `Link` component also needs to be able to receive tints? Then you wouldn't have to do this, right?

Reply:
Thanks for the tip. I put the Link in a TextBody component and could thus delete some custom CSS as the Link can inherit styles. 

 :ok: by @Kemosabert on 21/06/2018 13:13:19